nilearn/plotting/surface/_matplotlib_backend.py

```diff
@@ -745,6 +745,7 @@ def _plot_img_on_surf(
         symmetric_cbar = "auto"
     if cbar_tick_format is None:
         cbar_tick_format = "%i"
+    symmetric_cmap = kwargs.pop("symmetric_cmap", True)
```

Comment:
I'm a bit late to review this one, but this should set `None` instead of `True` because `symmetric_cmap` in not used in `plot_surf` for `matplotlib` backend.

Reply:
so it was an error that is was set to True below (line 812-813)?

Reply:
It was set to `True` in original code, as `plotly` and `matplotlib` code was mixed, and it is used only in `plotly`, not implemented for `matplotlib`. 

`matplotlib` did not give warning for parameters not implemented for that. I had changed that so that `matplotlib` also raises warning. I should have changed this too, Apparently I've missed that.

Reply:
> I can open a PR to fix that.

yup go for it

Reply:
I've rechecked the code: 

In the orginal code, `symmetric_cmap` is passed from `plot_img_on_surf` and `plot_surf_stat_map` to `plot_surf` as `symmetic_cmap=True` (If it was also set in `kwargs`, it could raise error).  It seems like, for `plotly` it was preferred to have `symmetric_cmap=True` for these two functions. 

However in `plot_surf` in `plotly`  engine, if `symmetric_cmap=None`, it is set to `False`. Is there a special reason for that? We could set it to `True` there. Then we won't need do any specific action to get `symmetric_cmap` from kwargs, it will be passed as it is.

Reply:
> Is there a special reason for that? We could set it to `True` there.

Not that I know

> Then we won't need do any specific action to get `symmetric_cmap` from kwargs, it will be passed as it is.

Then let's maybe do that: as long as the examples run fine and don't throw warnings

Reply:
By the way, the default for symmetric_cmap changed in plot_surf


None for the dev version of the doc

https://nilearn.github.io/dev/modules/generated/nilearn.plotting.plot_surf.html#nilearn.plotting.plot_surf

False for the stable version of the doc

https://nilearn.github.io/stable/modules/generated/nilearn.plotting.plot_surf.html#nilearn.plotting.plot_surf

It is OK because there is no change in actual default behavior, but this should be mentioned in the doc string of the function with a 

```rst
    .. versionchanged:: 0.12.0
        Default value changed to None.
```

The doc string should also explain what this value will take when using plotly.

Reply:
> Is it ok to change it to `True` instead of `False` for `plotly` in the backend and document it?

that would change the behavior in this case, no?